app/models/user.rb

```diff
@@ -12,4 +12,15 @@ class User < ApplicationRecord
   def downcase_email
     email.downcase!
   end
+
+  class << self
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost:
```

Comment:
```suggestion
      BCrypt::Password.create string, cost
```
Như này có chạy đc ko em